packages/governance/src/governParam.js

```diff
@@ -0,0 +1,107 @@
+// @ts-check
+
+import { details as X, q } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+import { Far } from '@agoric/marshal';
+
+import {
+  makeBallotSpec,
+  ChoiceMethod,
+  QuorumRule,
+  ElectionType,
+} from './ballotBuilder';
+import { assertType } from './paramManager';
+
+const paramChangePositions = (paramName, proposedValue) => {
+  const positive = `change ${paramName} to ${q(proposedValue)}.`;
+  const negative = `leave ${paramName} unchanged.`;
+  return { positive, negative };
+};
+
+/** @type {SetupGovernance} */
+const setupGovernance = async (
+  paramManagerAccessor,
+  registrarInstance,
```

Comment:
Is this the registrarInstance? I think it's the registrarCreator. The types in `SetupGovernance` seem slightly different than the parameters here.